### ISSUE TITLE
Preprocessing for finetuning the models

### ISSUE THREAD

**Issue by arunpatala** [flyyufelix/DenseNet-Keras#7]
do we need to do preprocssing of data like normalization and mean subtraction to fine tune 
 pretrained densenet models?
